core/settings.py

```diff
@@ -38,6 +38,8 @@ def get_env_variable(var_name, required=True, type=str):
 SITE_ROOT_URL = f"https://{SITE_HOST}"
 SECRET_KEY = get_env_variable("SECRET_KEY")
 DATAGOUV_API_KEY = get_env_variable("DATAGOUV_API_KEY", required=False)
+DATAGOUV_DOMAIN = "https://demo.data.gouv.fr"
```

Comment:
I need it to display the link to the dataset in the Mattermost report. But I could do otherwise, like providing only the production link, so I could put only the api url in settings. Tell me what you think about this.

Reply:
Ah yes I missed it, keep it that way it's fine